### PR TITLE
✨ Add github action support

### PR DESCRIPTION
**Pull request by daniel-rose**

### CODE REVIEW

**Review by julianhyatt — APPROVED** [review]

**Review by gengjozsef — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+on:
+  - pull_request
+
+jobs:
+  continous-integration:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: [ "7.3", "7.4" ]
```
Comment (SpiderDan98) [inline .github/workflows/main.yml:12]: Maybe php 8?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```
Comment (SpiderDan98) [inline .github/workflows/main.yml]: use checkout v3

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        uses: shivammathur/setup-php@v2 #https://github.com/shivammathur/setup-php
+        with:
+          php-version: ${{ matrix.php-versions }}
+          extensions: mbstring, dom, fileinfo, pgsql, json
+          coverage: xdebug #optional
+      - name: Get composer cache directory
+        id: composercache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - name: Cache composer dependencies
+        uses: actions/cache@v2
```
Comment (SpiderDan98) [inline .github/workflows/main.yml]: use cache v 3

**Review by SpiderDan98 — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (daniel-rose) [reply .github/workflows/main.yml:12]: I make a separate pr for this. But good point. Thx

**Pull request merged**